packages/plugins/rubygem-smart_proxy_omaha/rubygem-smart_proxy_omaha.spec

```
@@ -86,29 +107,31 @@ mkdir -p %{buildroot}%{content_dir}
 
 %files
 %dir %{gem_instdir}
-%config(noreplace) %attr(0640, root, %{proxy_user}) %{foreman_proxy_settingsd_dir}/omaha.yml
 %attr(-,%{proxy_user},%{proxy_user}) %{content_dir}
-%doc %{gem_instdir}/LICENSE
+%{_bindir}/smart-proxy-omaha-sync
+%config(noreplace) %attr(0644, root, root) %{_sysconfdir}/cron.d/%{name}
```

Also needs a change in the `mv` code above.
```suggestion
%config(noreplace) %attr(0644, root, root) %{_root_sysconfdir}/cron.d/%{name}
```